src/components/molecules/SecretPasswordForm/SecretPasswordForm.tsx

```diff
@@ -39,7 +38,7 @@ const SecretPasswordForm = ({ buttonText = "Join the party" }) => {
     async (e) => {
       e.preventDefault();
 
-      if (!isDefined(venueId)) {
+      if (!isDefined(spaceId)) {
```

Comment:
[OOS]
Should make `isNotDefined()` just so the `!` isn't used in places where it might be missed ( like between `(` and `i` )